### PR TITLE
Fix duplicate hamburger button

### PR DESCRIPTION
**Pull request by trangmei**
Fix https://github.com/Coursemology/coursemology2/issues/3065

Remove sidebar collapsed button from topbar
Add sidebar collapsed button to user row
Make user-sidebar flex

### CODE REVIEW

**Comment by fonglh** [comment]
Can post a screenshot? How are the menus after this goes in?

**Comment by trangmei** [comment]
image

File: .gitignore [hunk 1]
```diff
@@ -31,9 +31,6 @@
 /coverage/*
 /client/coverage/*
 
-# Ignore additional themes
```
Comment (fonglh) [inline .gitignore]: Leave this in. We don't change the default theme much and if you're working on additional themes you don't want to accidentally commit them to the main Coursemology repo.

Comment (jeremyyap) [reply .gitignore]: It is possible to include just the default folder though:

https://stackoverflow.com/questions/5533050/gitignore-exclude-folder-but-include-specific-subfolder

Comment (fonglh) [reply .gitignore]: Ok let's go with Jeremy's suggestion.

**Pull request merged**